tchannel/testing/vcr/patch.py

```diff
@@ -83,6 +85,7 @@ def send(self, arg1, arg2, arg3,
         vcr_request = VCRProxy.Request(
             serviceName=self.service.encode('utf-8'),
             hostPort=self.hostport,
+            knownPeers=self.original_tchannel.peers.hosts,
```

Comment:
I think this assumes its getting a `tchannel.tornado.TChannel`, can we make it also work with `tchannel.TChannel`?